README.md

```diff
@@ -73,8 +73,13 @@ in your executable path.
 With your [GOPATH](https://github.com/golang/go/wiki/GOPATH) already set up:
 
 ``` sh
-$ go get github.com/github/hub
-$ cd "$GOPATH"/src/github.com/github/hub
+$ mkdir -p "$GOPATH"/src/github.com/github
+$ cd "$GOPATH"/src/github.com/github
+$ git clone --config transfer.fsckobjects=false \
+            --config receive.fsckobjects=false \
+            --config fetch.fsckobjects=false \
```

Comment:
Since these instructions are getting to be a handful, people might want to copy/paste this whole block of operations. To help them with that, could we remove the leading `$` characters?

Reply:
ok!  I removed the `$` from other places in the file since, for consistency.